src/app/ui/recipes-list/recipes-list.component.ts

```diff
@@ -0,0 +1,25 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { Recipe } from "../../recipes/models/recipe";
+import { Subject } from "rxjs";
+import { MatListOption } from "@angular/material/list";
+
+@Component({
+  selector: 'app-recipes-list',
+  templateUrl: './recipes-list.component.html',
+  styleUrls: ['./recipes-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class RecipesListComponent {
+
+  @Input()
+  recipes: Recipe[] | null = [];
+
+  private selectionSubject = new Subject<Recipe>();
```

Comment:
We usually declare private properties with `_` prefix and observables with `$` suffix, so it should be named _selectionSubject$